scripts/create-fyre-stack.sh

```diff
@@ -146,7 +146,7 @@ parse_args() {
   fi
 
   if [[ -z "${VM_SIZE}" ]]; then
-    readonly VM_SIZE="s"
+    readonly VM_SIZE="m"
```

Comment:
Is this to resolve the kind test issue?

Reply:
Yes, small size VM generates a lot of timeout failures for Kind e2e test.